01-syntax/05-oop/Vector.js

```diff
@@ -1 +1,25 @@
-// Your code here.
+function Vector(x, y) {
```

Comment:
You need additional `eslint` props to pass `npm run lint`  
Take a look [here](https://github.com/nesterone/sf-eng-init-gl-spring-2016/blob/master/01-syntax/05-oop/Vector.js#L1)